### PR TITLE
Visually display scroll velocities, and measure more stably

### PR DESCRIPTION
**Pull request by gnprice**
This is the further scroll_overlay enhancement I mentioned at https://github.com/flutter/platform_tests/pull/13#issuecomment-1421313831 .

It produces a display that looks like this:

or this screen recording, from https://github.com/flutter/flutter/pull/120420#issuecomment-1425190476 :
<video src="https://user-images.githubusercontent.com/28173/218006707-5d9aff4e-8bb8-4001-ab6e-8df0a5429227.mp4">

This PR also includes several changes to make the measurements more stable. The numbers had been jumping around, which becomes a lot more visible with the visual display.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Review by Piinks — COMMENTED** [review]
Sharing on all 3 for consistency: I was so very excited to see https://github.com/flutter/platform_tests/pull/13 that I forgot myself and made an error, actually 2. I should have sought a second reviewer on the PR as is our review guidelines. And I should have ensured tests were included, or that test exemption had been granted. 😅

Can you add a test for this PR? It ensures we do not end up removing your change accidentally. :)

**Comment by gnprice** [comment]
> And I should have ensured tests were included, or that test exemption had been granted. :sweat_smile:
> 
> Can you add a test for this PR? It ensures we do not end up removing your change accidentally. :)

Does the test requirement apply to the platform_tests repo? From [the linked part of the wiki](https://github.com/flutter/flutter/wiki/Tree-hygiene#tests), it sounds like it may not:

> Every change in the flutter/engine, flutter/flutter, flutter/plugins, and flutter/packages repos must be tested, except for PRs that: […]

And there doesn't appear to be any existing tests for this app, or for `ios_widget_catalog_compare`. The third app in the repo, `tabs_overlay`, has a smoke test (which does seem like a good idea) but no more than that.

So it seems like the practice within this repo may be to mostly not have tests. Which I think is pretty reasonable, for the purpose these apps have, though a smoke test for each app would probably be helpful.

---

Oh, and I just spotted that that wiki section also says this:

> **A bot will comment on your PR if you need an explicit exemption.** Do not land a PR that has had such a message from the bot unless it has an explicit exemption from Hixie! The message tells you how to ask for an exemption (please don't just cc Hixie on the PR, he won't see it).

So I think that testing requirement really is meant to not apply to this repo, and it's just the PR template that is a bit misleading due to having been copied from the main repo.

(The template definitely hasn't been carefully adjusted to fit this repo, because some of the other items don't really fit either: e.g. Tree Hygiene, as I think this repo isn't part of the Flutter build.)

**Comment by Piinks** [comment]
Yeah I too noticed the inconsistencies, I am following up with the infra team to see if our bot just overlooks this repo intentionally or not. The bot ensures multiple compliances are met before code is merged, so it could be that it really should be enabled here.

It is always good to have a test that prevents your change from being reverted, perhaps a smoke test will suffice here to make sure it still runs. I have in the past returned to this tool and found it broken. 

I'll follow up with what I find out about the template/bot mix up here. https://discord.com/channels/608014603317936148/608021351567065092/1075106318538518558

**Comment by gnprice** [comment]
Fair enough. I'll add a quick test.

**Comment by gnprice** [comment]
Added a test for the new VelocityOverlay widget; PTAL.

Also fixed up some things that were needed in order for it to work in tests. One was pretty puzzling:
7dcd87655 Add Expanded on number labels
but I found a reasonable enough workaround.

Another was telling me that the `velocityTimer` member was no longer needed after these changes and can be removed:
749440d8c Cut now-unused velocityTimer
so that was a nice small cleanup.

**Review by Piinks — APPROVED** [review]
Thanks for yak shaving! This also LGTM, but there are merge conflicts from the other PR landing

**Comment by gnprice** [comment]
Great, thanks!

Will rebase and resolve those conflicts, here and on #16.

**Comment by gnprice** [comment]
Done.

**Pull request merged**
